qt/widgets/common/src/ManageUserDirectories.cpp

```diff
@@ -125,13 +159,18 @@ void ManageUserDirectories::loadProperties() {
   }
 
   // default save directory
-  QString saveDir = QString::fromStdString(
-                        Mantid::Kernel::ConfigService::Instance().getString(
-                            "defaultsave.directory"))
-                        .trimmed();
+  QString saveDir =
```

Comment:
Can this be `const auto`?

Reply:
Thanks! I think I got bored of changing from QString->auto. I'll get them updated.